product-development/README.md

```diff
@@ -33,6 +33,35 @@ Anyone advocating for an epic's priority (or priorities in general) should @-men
 
 <img src="images/epics-priority.png" height="100" />
 
+### Estimating Stories
+
+Right now we have Clubhouse configured to use the following, roughly Fibonacci sequence story point estimate scale:
+
+* 1
+* 2
+* 3
+* 5
+* 8
+* 13
+* 21
+
+We use story points for one purpose: to communicate with each other about the complexity of work. As an engineer this is useful when it's 4 PM and I'm looking at an epic I'm working on it's helpful to know if a story is _expected_ to be simple or complicated. As a PM/EM/CEO this is useful when I want to ask questions like "roughly how much work is left on X" and "when should we start work on marketing for Y".
+
+Put another way: we use story points to communicate expectations, to our peers, and to those who operate at a higher level in the company.
+
+#### What do these numbers mean?
+Each of these is for one engineer. For example, a 1 point story is a story that we expect an engineer would be able to complete in half of one day.
```

Comment:
So glad you included this section! Might be worth noting that the same task might have different point estimates depending on which engineer is working on it, and empower an eng to adjust the point estimate for themselves after taking a ticket--but point estimates will still be useful for order of magnitude even if they vary between engineers.

Also maybe worth noting somewhere that when you take on an epic / when reviewing someone else's epic, it's good to gut-check point estimates and raise any concerns you have--this will help us uncover scope disagreements, hidden edge cases etc. early.